include/boost/geometry/strategies/area/cartesian.hpp

```diff
@@ -49,6 +49,7 @@ struct strategy_converter<strategy::area::cartesian<CT> >
 {
     static auto get(strategy::area::cartesian<CT> const& strategy)
     {
+        boost::ignore_unused(strategy);
```

Comment:
Here the parameter name could be removed instead, but `ignore_unused` works too.